gdb/stl-views.gdb

```diff
@@ -1,11 +1,17 @@
-#                                                                                                        
+# This file is part of the dune-stuff project:
+#   https://github.com/wwu-numerik/dune-stuff
+# The copyright lies with the authors of this file (see below).
+# License: BSD 2-Clause License (http://opensource.org/licenses/BSD-2-Clause)
+# Authors:
+#   Sven Kaulmann (2010)
+#
 #   STL GDB evaluators/views/utilities - 1.03
 #
-#   The new GDB commands:                                                         
-# 	    are entirely non instrumental                                             
+#   The new GDB commands:
+# 	    are entirely non instrumental
```

Comment:
These are probably whitespace problems in the original file, the fixes of which could be commited beforehand (if we care).